content/en/docs/quickstart.md

```diff
@@ -29,19 +32,36 @@ docker build -t localhost:5000/net-monitor:v1 https://github.com/wabbit-networks
 docker push localhost:5000/net-monitor:v1
 ```
 
+Tags are mutable and a tag reference can point to a different container image than the one signed. Find out the digest value from the output of `docker push` command.
+
+An example output of `docker push`:
+
+```output
+The push refers to repository [localhost:5000/net-monitor]
+2556c54bfdf3: Pushed
+fb6ca4f9c8d3: Pushed
+ded7a220bb05: Pushed
+v1: digest: sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a size: 942
+```
+
+Use digest to identify the container image as `localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a`.
+
+> Note：
+> Tags can be used to reference the container image. In this example, you can use tag `v1` for reference. Notation commands will resolve the tag `v1` to digest `sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a` first. By validating the digest, you can make sure it is the right container image to reference.
+
 ## List the signatures associated with the container image
 
 Use `notation list` to show any signatures associated with the container image you built and pushed in the previous section.
 
 ```console
-notation list $IMAGE
+notation list localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
```

Comment:
```suggestion
IMAGE=localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
notation list $IMAGE
```

Consider using the variable by setting it once and then using it throughout. It looks cleaner and is easier for users to copy/paste. With the current changes, they must update every command